src/components/OAuthLogoutButton.tsx

```diff
@@ -1,20 +1,33 @@
 import React, { FC, useCallback } from 'react';
-import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
+import {
+  TouchableOpacity,
+  TouchableOpacityProps,
+  View,
+  Text,
+  ViewStyle,
+  TextStyle,
+  Platform,
+} from 'react-native';
+import { useStyles } from '../hooks/useStyles';
 import { tID } from '../common/testID';
 import { useAuth } from '../hooks/useAuth';
 import { LogoutParams, useOAuthFlow } from '../hooks/useOAuthFlow';
+import { Theme } from './BrandConfigProvider/theme/Theme';
 
 type OAuthLogoutButtonParams = Omit<TouchableOpacityProps, 'onPress'> &
   LogoutParams & {
-    children?: React.ReactNode;
+    label: string;
+    styles?: OAuthLogoutButtonStyles;
   };
 
 export const OAuthLogoutButton: FC<OAuthLogoutButtonParams> = ({
   onSuccess,
   onFail,
-  children,
+  label,
+  styles,
   ...touchableOpacityProps
 }) => {
+  const { styles: $ } = useStyles('OAuthLoginButton', defaultStyles, styles);
```

Comment:
Why does `useStyles` return an object? Are we planning on returning something more than just the `styles` property?

Reply:
Why are we calling the var `$` ?

Reply:
> Why does useStyles return an object?

Just leaving the API open to return something else as well.  If/when we need to return something else, this makes it easier to change.  I'm open to doing it differently however.  Any reason not to return an object here?

> Why are we calling the var `$` ? 

This is a convention I'm proposing we adopt to solve two problems.  Completely open to doing it differently, but here is my thinking:

There is a collision with just using `styles` because of the instance styles prop being passed into the component.  I do like `instanceStyles` here:  `useStyles('OAuthLoginButton', defaultStyles, instanceStyles)`, however I prefer keeping the public API of the component simple and straightforward with just `styles` like this: `<OAuthButton styles={} />`

I'm suggesting using the `$` because I think it will be a commonly used symbol internally throughout all our components and instantly communicates that this is a style object, and not just any style object but the final, merged style reduced from many possible locations.

This idea is similar to how [Ignite does styling](https://github.com/infinitered/ignite/blob/master/docs/Styling.md).  While Ignite _is_ the most popular React Native project generator, I don't know if I would go so far as to say this is a "standard convention".  Therefore I can't defend it as such.  However, I can say that I think it should _become_ a standard convention.  I've grown to like how it makes the declarative TSX syntax easier to read by instantly identifying all style variables and removing duplication.  I know it is dangerously close to the horrible practice of using single letter variable names, but this is an exception that proves the rule because it is a quick onboard for anyone who is seeing it for the first time.  They don't have to figure it out each time in different contexts.  

Also, it seems audacious, and I'm wearing my LifeOmic Audacious t-shirt today.  Maybe it's time we take the `$` back from jQuery.  😄   

Obviously this isn't the only way to solve this. Another way to do it would be to name it `mergedStyles`.  Here is the difference:

![Screenshot 2023-03-06 at 10 55 20 AM](https://user-images.githubusercontent.com/220893/223163106-e3a10b14-b0cf-409d-857b-c271c1fd6e94.png)

vs

![Screenshot 2023-03-06 at 10 56 39 AM](https://user-images.githubusercontent.com/220893/223163165-00895ea3-bc21-45d4-80ef-e70e33b1c3e1.png)

Anyways, I don't want to do this if we're not all on board, so let me know what you think, I won't be offended.  :-)

Reply:
Updated this to remove `$` and use `instanceStyles`.  👍